modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple clusters, thus helping to avoid a single point of failure. 
+
+[NOTE]
+====
+{product-title} vSphere topology awareness does not support multiple vCenter datacenters.
+====
+
+This is accomplished by defining zone and region categories in vCenter, and then assigning these categories to different failure domains, such as a compute cluster or host, by creating tags for these zone and region categories. After you have created the appropriate categories, and assigned tags to vCenter objects, you can create additional machinesets that create virtual machines (VMs) that are responsible for scheduling pods in those failure domains.
+
+.Procedure
+. In the VMware vCenter vSphere client GUI, define appropriate zone and region catagories and tags.
++
+While vSphere allows you to create categories with any arbitrary name, {product-title} strongly recommends use of `openshift-region`` and `openshift-zone`` names for defining topology.
```

Comment:
Here strings like `openshfit-region` etc seems double escaped with backticks. They should be only once escaped.

Reply:
Done.